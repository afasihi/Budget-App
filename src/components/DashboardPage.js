import React from 'react';
import ExpensesList from './ExpensesList';
import ExpenseListFilters from './ExpenseListFilters'
import ExpensesSummery from './ExpensesSummery'

const DashboardPage = () => (
	<div>
		<ExpensesSummery />
		<ExpenseListFilters />
		<ExpensesList />
	</div>
)

export default DashboardPage;