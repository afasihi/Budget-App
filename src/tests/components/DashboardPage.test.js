import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/DashboardPage';

test('Should render the DashboardPage correctly', () => {
	const wrapper = shallow(<DashboardPage/>)
	expect(wrapper).toMatchSnapshot();
})