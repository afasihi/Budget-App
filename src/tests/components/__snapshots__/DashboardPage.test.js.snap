// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the DashboardPage correctly 1`] = `
<div>
  <Connect(ExpensesSummery) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
